src/components/text-editor/text-editor.tsx

```diff
@@ -93,6 +95,9 @@ export class TextEditor implements FormComponent<string> {
     @Prop({ reflect: true })
     public value: string;
 
+    @Prop()
```

Comment:
Perhaps this should be marked as private as well? :slightly_smiling_face:

Reply:
Yes :)